larva/src/test/java/nl/nn/adapterframework/testtool/TestToolTest.java

```diff
@@ -37,9 +37,8 @@ public void decodeUnzipContentBetweenKeysFromIgnoreMap() {
         assertEquals(identifier.get("key2"), key2);
         assertEquals(identifier.get("replaceNewlines"), replaceNewlines);
     }
-    
-    @Test
```

Comment:
Graag wel een lege regel tussen de methodes laten staan. Of staat die er nu nog wel?